tests/unit/static-build-loader/getFeatures.ts

```diff
@@ -52,6 +54,7 @@ registerSuite('getFeatures', {
 		assert.deepEqual(getFeatures(['ie11', 'node']), {
 			arraybuffer: true,
 			blob: true,
+			'dom-intersection-observer': false,
```

Comment:
do you need `dom-pointer-events` here as well?

Reply:
i had it there initially and the tests were failing as a result. i imagine it's because it's false for both?